src/pages/workspace/members/WorkspaceMemberNewCardPage.tsx

```diff
@@ -64,11 +68,19 @@ function WorkspaceMemberNewCardPage({route, personalDetails}: WorkspaceMemberNew
             });
             Navigation.navigate(ROUTES.WORKSPACE_EXPENSIFY_CARD_ISSUE_NEW.getRoute(policyID, ROUTES.WORKSPACE_MEMBER_DETAILS.getRoute(policyID, accountID)));
         } else {
+            const data: Partial<AssignCardData> = {
+                email: memberLogin,
```

Comment:
NAB: Let's add bankName value here

Reply:
thanks, wouldn't say that is a NAB, I removed setting the feed in the assign card flow

Reply:
@koko57 It is a NAB because we already saved feed on the route. But I think we still need to fix it

Reply:
yeah, you're saving the feed on the route but not in the Onyx

Reply:
@koko57  Let's add `cardName` along with `email`